includes/wp-cli.php

```diff
@@ -163,15 +165,252 @@ function import_from_csv( $args, $assoc_args ) {
 				WPCOM_Legacy_Redirector::insert_legacy_redirect( $redirect_from, $redirect_to );
 
 				if ( 0 == $row % 100 ) {
-					if ( function_exists( 'stop_the_insanity' ) )
+					if ( function_exists( 'stop_the_insanity' ) ) {
 						stop_the_insanity();
+					}
 					sleep( 1 );
 				}
 			}
 			fclose( $handle );
 		}
 	}
 
+	/**
+	 * Verify our redirects.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--status=<status>]:
+	 * : Filter by verification status.
+	 * ---
+	 * default: unverified
+	 * options:
+	 *   - unverified
+	 *   - verified
+	 *   - all
+	 * ---
+	 *
+	 * [--format=<format>]
+	 * : Render output in a particular format.
+	 * ---
+	 * default: csv
+	 * options:
+	 *   - table
+	 *   - json
+	 *   - yaml
+	 *   - csv
+	 * ---
+	 *
+	 * [--verbose]
+	 * : Print notifications to the console for passing URLs.
+	 *
+	 * [--strict]
+	 * : Enable verification for validated redirects to post ids.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * wp wpcom-legacy-redirector verify-redirects
+	 *
+	 * @subcommand verify-redirects
+	 * @synopsis [--status=<status>] [--format=<format>] [--verbose] [--strict]
+	 */
+	function verify_redirects( $args, $assoc_args ) {
```

Comment:
The code for this command is starting to feel really long and I'm wondering if we should break it down into smaller methods?

Reply:
my last commit moved the validation & verification processes into their own functions, leaving the main loops (which are tied to the cli progressmeter) in the main command.

do you have any suggestions for if/how this could be further broken down into separate methods?